blog/en/blog/2024/08/01/free-tier-api-apisix.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Free tier API with Apache APISIX
+authors:
+  - name: Nicolas Fränkel
+    title: Author
+    url: https://github.com/nfrankel
+    image_url: https://avatars.githubusercontent.com/u/752258
+keywords:
+  - APISIX
+  - Free tier
+  - Rate limiting
+  - Consumer
+description: >
+  Lots of service providers offer a free tier of their service. The idea is to let you kick their service's tires freely. If you need to go above the free tier at any point, you'll likely stay on the service and pay. In this day and age, most services are online and accessible via an API. Today, we will implement a free tier with Apache APISIX.
```

Comment:
> implement a free tier with Apache APISIX

I am a bit confused with this wording. Usually you would say developers use the free tier services to implement something. Are you writing from the perspective of a MSP?

Reply:
I am. The wording seems to me quite clear:

>Today, we will implement a free tier with Apache APISIX.